src/Symfony/Component/Dotenv/Dotenv.php

```diff
@@ -50,6 +51,33 @@ public function load(string $path, string ...$paths): void
         $this->doLoad(false, $path, $paths);
     }
 
+    /**
+     * Loads one or several .env and the corresponding env.$env, env.local and env.$env.local files if they exist.
+     *
+     * .env.local is always ignored in test env because tests should produce the same results for everyone.
+     *
+     * @param string    $path  A file to load
+     * @param ...string $paths A list of additional files to load
+     *
+     * @throws FormatException when a file has a syntax error
+     * @throws PathException   when a file does not exist or is not readable
+     *
+     * @see https://github.com/bkeepers/dotenv#what-other-env-files-can-i-use
+     */
+    public function loadForEnv(string $env, string $path, string ...$paths): void
```

Comment:
I'd suggest renaming $paths to $extraPaths everywhere in this PR

Reply:
done